src/views/common.tsx

```diff
@@ -9,9 +15,16 @@ export const Page = (props: Html.PropsWithChildren<{ title: string }>): JSX.Elem
         <script src="lib/htmx.org/ext/json-enc.js"></script>
         <link rel="icon" type="image/ico" sizes="48x48" href="/public/images/favicon.ico" />
         <link rel="stylesheet" type="text/css" href="/public/styles/main.css" />
+        <link rel="stylesheet" type="text/css" href="/public/styles/semantic.css" />
```

Comment:
does this exist?

Reply:
nope, tested something earlier another css for some basics e.g. layout